### PR TITLE
this is learning website

### PR DESCRIPTION
**Pull request by haroon786**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -170,26 +169,27 @@
...
-    "navigation": {
+
+    "navbar": {
       "projectType": "library",
-      "root": "libs/navigation",
-      "sourceRoot": "libs/navigation/src",
-      "prefix": "til",
+      "root": "libs/navbar",
+      "sourceRoot": "libs/navbar/src",
+      "prefix": "this-is-learning",
```
Comment (santoshyadavdev) [inline angular.json]: keep this til

File: apps/this-is-learning/src/styles.scss [hunk 2]
```diff
@@ -1,4 +1,8 @@
 /* You can add global styles to this file, and also import other style files */
 @import 'tailwindcss/base';
 @import 'tailwindcss/components';
-@import 'tailwindcss/utilities';
\ No newline at end of file
+@import 'tailwindcss/utilities';
```
Comment (santoshyadavdev) [inline apps/this-is-learning/src/styles.scss]: can you remove extra space, or run the prettier

File: libs/navbar/.eslintrc.json [hunk 3]
```diff
@@ -13,15 +13,15 @@
           "error",
           {
             "type": "attribute",
-            "prefix": "til",
+            "prefix": "thisIsLearning",
```
Comment (santoshyadavdev) [inline libs/navbar/.eslintrc.json]: keep til

File: libs/navbar/.eslintrc.json [hunk 4]
```diff
@@ -13,15 +13,15 @@
...
             "style": "camelCase"
           }
         ],
         "@angular-eslint/component-selector": [
           "error",
           {
             "type": "element",
-            "prefix": "til",
+            "prefix": "this-is-learning",
```
Comment (santoshyadavdev) [inline libs/navbar/.eslintrc.json]: same here

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]
Can you run prettier for all the files.

**Review by santoshyadavdev — APPROVED** [review]
thank you :)

**Review by santoshyadavdev — CHANGES_REQUESTED** [review]
Please check the prevois comments

**Review by santoshyadavdev — APPROVED** [review]
Thank you for making changes

File: apps/this-is-learning-e2e/src/integration/app.spec.ts [hunk 5]
```diff
@@ -3,11 +3,11 @@ import { getGreeting } from '../support/app.po';
...
-    // Example: cy.login('my-email@something.com', 'myPassword');
+  // it('should display welcome message', () => {
+  //   // Custom command example, see `../support/commands.ts` file
+  //   // Example: cy.login('my-email@something.com', 'myPassword');
 
-    // Function helper example, see `../support/app.po.ts` file
-    getGreeting().contains('Hello, Scully!');
-  });
+  //   // Function helper example, see `../support/app.po.ts` file
+  //   getGreeting().contains('Hello, Scully!');
```
Comment (LayZeeDK) [inline apps/this-is-learning-e2e/src/integration/app.spec.ts]: issue: Please keep at least one working end-to-end test (we're using Cypress)

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
The end to end test is now failing, Haroon

**Comment by haroon786** [comment]
unfortunately, I have never written any test cases :(, now  it passed all checks

**Review by santoshyadavdev — APPROVED** [review]

**Review by LayZeeDK — CHANGES_REQUESTED** [review]
issue: Please consider whether we are directly using the `lightBlue` Tailwind variable
```
warn - As of Tailwind CSS v2.2, `lightBlue` has been renamed to `sky`.
warn - Please update your color palette to eliminate this warning.
```

**Review by LayZeeDK — APPROVED** [review]

**Pull request merged**